lib/publiccloud/basetest.pm

```diff
@@ -71,27 +71,27 @@ sub provider_factory {
         $args{service} //= 'GCE';
         if ($args{service} eq 'GCR') {
             $provider = publiccloud::gcr->new(
-                account => get_var('PUBLIC_CLOUD_ACCOUNT'),
-                service_acount_name => get_var('PUBLIC_CLOUD_SERVICE_ACCOUNT'),
-                project_id => get_var('PUBLIC_CLOUD_PROJECT_ID'),
+                account => get_var('PUBLIC_CLOUD_GCP_ACCOUNT'),
+                service_acount_name => get_var('PUBLIC_CLOUD_GCP_SERVICE_ACCOUNT'),
+                project_id => get_var('PUBLIC_CLOUD_GCP_PROJECT_ID'),
                 private_key_id => get_var('PUBLIC_CLOUD_KEY_ID'),
                 private_key => get_var('PUBLIC_CLOUD_KEY'),
-                client_id => get_var('PUBLIC_CLOUD_CLIENT_ID'),
+                client_id => get_var('PUBLIC_CLOUD_GCP_CLIENT_ID'),
                 region => get_var('PUBLIC_CLOUD_REGION', 'europe-west1-b'),
-                storage_name => get_var('PUBLIC_CLOUD_STORAGE', 'openqa-storage'),
+                storage_name => get_var('PUBLIC_CLOUD_GCP_STORAGE', 'openqa-storage'),
                 username => get_var('PUBLIC_CLOUD_USER', 'susetest')
             );
         }
         elsif ($args{service} eq 'GCE') {
             $provider = publiccloud::gce->new(
-                account => get_var('PUBLIC_CLOUD_ACCOUNT'),
-                service_acount_name => get_var('PUBLIC_CLOUD_SERVICE_ACCOUNT'),
-                project_id => get_var('PUBLIC_CLOUD_PROJECT_ID'),
+                account => get_var('PUBLIC_CLOUD_GCP_ACCOUNT'),
```

Comment:
Here we're in *GCE* but the variables contain *GCP* plus on line 72 we can also see *GCR*.
Honestly don't follow this any more. What about `PUBLIC_CLOUD_GOOGLE_` instead?

Reply:
ok... I agree that in this way is more clear

Reply:
Done